README.md

```diff
@@ -12,10 +12,28 @@ Swift toolchain version 5.4 or greater is required.
 
 The `main` branch depends on SwiftPM's `main` branch and may be unstable. It is recommended to use versioned branches such as `5.5`, which depend on the corresponding SwiftPM's `release/<version>` branch, instead.
 
+
+## Installation
+
+Currently, the package collection generator is a standalone tool that's not integrated with the Swift toolchain. To use it from the command line, first build the project from source by cloning the repository and running the following from the root directory:
+
+```zsh
+swift build --configuration release
+```
+
+Then, install the final executables on your system path:
+
+```zsh
+install .build/release/package-collection-generate /usr/local/bin/package-collection-generate
+install .build/release/package-collection-diff /usr/local/bin/package-collection-diff
+install .build/release/package-collection-sign /usr/local/bin/package-collection-sign
+install .build/release/package-collection-validate /usr/local/bin/package-collection-validate
+```
+
```

Comment:
Please add:

Finally, another way to run the tool is via `swift run`, which builds and runs the specified executable. For example,

```zsh
swift run package-collection-generate
```

This might be useful for developers to test out local code changes.